.github/workflows/android-custom-build.yml

```diff
@@ -22,17 +22,11 @@ jobs:
       - name: Install npm dependencies
         run: |
           npm install
-      - name: Patch selected backend service URL
-        run: |
-          ESCAPED_BACKEND_SERVICE_URL=$(printf '%s\n' "$BACKEND_SERVICE_URL" | sed -e 's/[\/&]/\\&/g')
-          ESCAPED_BACKEND_SERVICE_DEFAULT_URL=$(printf '%s\n' "${{ env.backendServiceDefaultUrl }}" | sed -e 's/[\/&]/\\&/g')
-          echo "Selected backend service URL: $BACKEND_SERVICE_URL (escaped: $ESCAPED_BACKEND_SERVICE_URL)"
-          sed -i "s/$ESCAPED_BACKEND_SERVICE_DEFAULT_URL/$ESCAPED_BACKEND_SERVICE_URL/g" shared/request.ts
-        env:
-          BACKEND_SERVICE_URL: ${{ github.event.inputs.backendServiceUrl }}
       - name: Build App Newlogic Release
         run: |
           cd android && ./gradlew :app:assembleNewlogicRelease
+        env:
+          BACKEND_SERVICE_URL: ${{ github.event.inputs.backendServiceUrl }}
```

Comment:
> @pmigueld we should remove the on.workflow_dispatch.inputs sections from the workflow file as it is not used anymore.
@uocnb isn't it used here?

Reply:
Oh, sorry, my fault. I thought it was used input field instead of choice. 
We should use `type: string` instead to allow user to enter custom value when workflow start, because the other available URL is not finalized yet. [See sample here](https://docs.github.com/en/actions/learn-github-actions/contexts#example-usage-of-the-inputs-context).